tap_github/client.py

```diff
@@ -188,3 +189,41 @@ def parse_response(self, response: requests.Response) -> Iterable[dict]:
             results = [resp_json]
 
         yield from results
+
+
+class GitHubGraphqlStream(GraphQLStream, GitHubRestStream):
+    """GitHub Graphql stream class."""
+
+    @property
+    def url_base(self) -> str:
+        return f'{self.config.get("api_url_base", self.DEFAULT_API_BASE_URL)}/graphql'
+
+    # the path in dot notation under which to fetch the list of
+    # records from the graphql response
+    query_path: str = ""
+
+    def parse_response(self, response: requests.Response) -> Iterable[dict]:
+        """Parse the response and return an iterator of result rows.
+
+        Args:
+            response: A raw `requests.Response`_ object.
+
+        Yields:
+            One item for every item found in the response.
+
+        .. _requests.Response:
+            https://docs.python-requests.org/en/latest/api/#requests.Response
+        """
+        resp_json = response.json()
+        data = glom(resp_json["data"], self.query_path)
```

Comment:
TIL `glom` 🙂 

The SDK already supports [JSONPath](https://sdk.meltano.com/en/latest/dev_guide.html?highlight=jsonpath#restful-jsonpaths), so I'm curious what are the benefits of glom over that.

Reply:
@edgarrmondragon I am trying to use JSONPath and it works with online tools but I can't get it to work in the code.

This is what I am doing:

resp_json = response.json()
yield from extract_jsonpath(self.query_jsonpath, input=resp_json)

with `query_jsonpath = "$.data.user.repositoriesContributedTo.nodes.*"`

Reply:
nvm I think I fixed it in https://github.com/MeltanoLabs/tap-github/pull/78/commits/9435df7addb65237d375f543ba83f626a214456c

Reply:
@ericboucher looks good!